### PR TITLE
Capstone SmartEd Online PR

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this PR I have : 
- added the mobile version
- added the desktop version 
- implement courses with JavaScript
- Added linters
- used Bootstrap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,238 @@
...
+
+
+show me your support by following me on github and giving me a star ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+This project was designed to test some of the abilities I learned in the first module of the Microverse program. I had to create a website from scratch following basic [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
```
Comment (Ibizugbe) [inline README.md]: Nice job documenting your project👍 Just as requested in the requirements, you should give some acknowledgment to the author of the design [Cindy Shin](https://www.behance.net/adagio07)😉. Kindly do well to give the author of the design some credit👍

cindy ackno

File: index.html [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                  </div>
+                </div>
+                <div class="col-md-3 ">
+                    <div class="feat-box p-3">
+                      <div class="feat-img d-flex justify-content-center">
+                        <img src="images/icons/5.png" alt="">
+                      </div>
+                      <h3 class="text-justify text-center pt-3">Advanced Evaluation</h3>
+                      <p class="text-justify text-center pt-2"> Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur.</p>
+                    </div>
```
Comment (Ibizugbe) [inline index.html]: Nice job adding the cards for the course feature section😉 but let's ensure to stick to the design by making the cards 5 in number as shown in the design👍 as it is required that we stick to the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) but personalize it😉

image

Kindly add one more card to the section to make your project match the[ design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)👍.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                      <img src="images/icons/2.png" alt="">
+                    </div>
+                    <h3 class="text-justify text-center pt-3 ">Master Classes</h3>
+                    <p class="text-justify text-center pt-2"> Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur.</p>
+                  </div>
+                </div>
+                <div class="col-md-3 ">
+                    <div class="feat-box p-3">
+                      <div class="feat-img d-flex justify-content-center">
+                        <img src="images/icons/5.png" alt="">
```
Comment (Ibizugbe) [inline index.html]: Kindly ensure every image has an `alt` value as this gives your project a high accessibility score.😉 I suggest you add a descriptive value to every `alt` attribute on your project👍

File: README.md [hunk 4]
```diff
@@ -1 +1,238 @@
...
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/rachaouldbabaali/capstone-1/blob/capstone/MIT.md) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+=======
+>>>>>>> main
```
Comment (Ibizugbe) [inline README.md]: I suggest you take away this `merge conflict` text from your readme as it gives it a less professional look👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali 👋👋👋, 

tryagain


Good job so far!👍👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] Github flow practice have been used👍.
- [x] An array of object have been created and the speaker section is rendered dynamically using the data from the array of object👍.
- [x] UI looks very impressive👍.
- [x] A descriptive video have been created to walk the user through the project👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Kindly ensure your commit messages are always descriptive with every message being capitalized👍.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @rachaouldbabaali 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**